.travis.yml

```diff
@@ -77,6 +77,7 @@ after_success:
     if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then
       # macpython 3.5 doesn't support recent TLS protocols which causes twine
       # upload to fail, so we use the system Python to run twine
+      /usr/bin/python -m ensurepip
```

Comment:
I'm pretty sure [Travis doesn't run this script as root](https://travis-ci.org/zopefoundation/zope.interface/jobs/610352400#L86), so [all of the install commands](https://github.com/zopefoundation/BTrees/issues/113#issuecomment-552103774) need the `--user` flag. I'm not sure that the user directory is on the default path, so I think the rest of the commands need modified to reference the module:

```
$ /usr/bin/python -mensurepip --user
$ /usr/bin/python -m pip install -U --user pip
$ /usr/bin/python -m pip install -U --user twine
$ /usr/bin/python -m twine upload --skip-existing dist/*
```

Reply:
(The reason I'm floundering is I hate pip install --user, no rational reason.)